crossdock-go/crossdock/doc.go

```diff
@@ -34,4 +34,5 @@
 // 	entries := behavior.Run(func(s Sink) {
 // 		MyBehavior(s, someParams)
 // 	})
-package behavior
+
```

Comment:
Delete this empty line. The blob above is the package documentation so it should be attached to the `package` declaration.